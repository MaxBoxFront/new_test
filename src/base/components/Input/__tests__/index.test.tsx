import React from 'react';
import renderer from 'react-test-renderer';

import { Input } from 'base/components/Input';

describe('The Input component', () => {
	it('should renders correctly', () => {
		const tsx = (<Input name="email"
												value=''
												label=""
												placeholder="Email"
												onChange={() => {}}
												fluid />);
		const tree = renderer
			.create(tsx)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
