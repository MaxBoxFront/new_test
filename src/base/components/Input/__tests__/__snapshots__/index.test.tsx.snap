// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Input component should renders correctly 1`] = `
<div
  className="input__wrapper"
>
  <input
    className="input input--fluid"
    id="email"
    onChange={[Function]}
    placeholder="Email"
    type="text"
    value=""
  />
</div>
`;
